cpp/src/slice2swift/SwiftUtil.h

```diff
@@ -8,94 +8,78 @@
 
 using StringPairList = std::list<std::pair<std::string, std::string>>;
 
-namespace Slice
+namespace Slice::Swift
 {
-    const int TypeContextInParam = 1;
-    const int TypeContextProtocol = 2;
```

Comment:
These were only used alongside the `typeCtx` parameter of `writeMembers`.
That parameter was removed, so these constants are dead now.